src/Couchbase.Lite.Shared/Sync/ReplicatorOptionsDictionary.cs

```diff
@@ -175,8 +176,14 @@ public string Filter
         /// by a <see cref="Replicator"/> with this configuration will be rejected.
         /// </summary>
         [CanBeNull]
-        public X509Certificate2 PinnedServerCertificate { get; set; }
+        internal X509Certificate2 PinnedServerCertificate { get; set; }
```

Comment:
Why did this change to internal?

Reply:
There is a public facing API in Replicator config, so I mark this one internal. (I know it will not make any difference since the class itself is internal.) I can change it back. But I will keep NetworkInterface here as internal...

Reply:
It's mostly a style choice then, so either way is fine.